packaging/hydra.nix

```
@@ -196,15 +187,15 @@ in
     let pkgs = nixpkgsFor.${system}.native; in
     pkgs.runCommand "install-tests"
       {
-        againstSelf = testNixVersions pkgs pkgs.nix pkgs.pkgs.nix;
+        againstSelf = testNixVersions pkgs pkgs.pkgs.nix;
```

```suggestion
        againstSelf = testNixVersions pkgs pkgs.nix;
```

Halloween is over. If this isn't the same, I'm not sure of that.